### PR TITLE
Adding pngx to producer

### PR DESCRIPTION
**Pull request by mdiplacido**
Getting PNGX to be supported end2end.  the windows producer app now just writes pngx format to the file and sftp streams.   Also fixed a contrast/border bug with the way we rendered the QRCode which made it hard for some QR apps to read the QR.

### CODE REVIEW

File: kiosk-browser-client/src/App.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ class App extends React.Component<any, IState> {
           <div className={`qr-box ${this.state.showQR ? 'open' : 'close'}`}>
             {
               this.state.hasQR && this.state.currentImage && this.state.currentImage.url ? 
-              (<QRCode value={this.state.currentImage.url} renderAs="svg" />) : 
+              (<QRCode value={this.state.currentImage.url} renderAs="svg" fgColor="#0078d4" bgColor="#fff" />) :
```
Comment (mdiplacido) [inline kiosk-browser-client/src/App.tsx]: we needed more contrast and a discernible border (that's achieved through the CSS).

File: package.json [hunk 2]
```diff
@@ -4,10 +4,13 @@
   "description": "Top Level piosk node packages",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "install:client": "cd kiosk-browser-client && npm install",
```
Comment (mdiplacido) [inline package.json]: making it easier to install from the root of the workspace.

File: kiosk-browser-client/src/App.tsx (hunk 1 above)
Comment (junjue) [reply kiosk-browser-client/src/App.tsx]: i'm okay with the contrast change!
and there's an attr available for border if you need, includeMargin = true

Comment (mdiplacido) [reply kiosk-browser-client/src/App.tsx]: cool.  i just tried this and it turns out our typing info for this component does not have this attribute, but i see the test site associated with the component does have this.   what i have does something similar, lets just go with this for now.

**Pull request merged**